src/components/nav-drawer/styles/nav-drawer.base.scss

```
@@ -28,6 +26,36 @@
     [part='mini'] {
         width: rem(60px);
     }
+
+    [part='overlay'] {
+        opacity: .54;
+        background: color(gray, 500);
+        position: fixed;
+        top: 0;
+        bottom: 0;
+        left: 0;
+        right: 0;
+        z-index: 2;
+    }
+}
+
+:host([position='start']) [part='mini'],
+:host([position='end']) [part='mini'] {
+    background: color(surface, 500);
+    box-shadow: var(--igc-elevation-16);
+    position: fixed;
+    top: 0;
+    bottom: 0;
+}
+
+:host([position='start']) [part='mini'] {
+    border-inline-end: 1px solid color(gray, 900, .14);
+    left: 0;
```

Make sure to use the appropriate direction-agnostic properties so that RTL is supported out of the box.

```suggestion
    inset-inline-start: 0;
```